packages/generate/src/cli.ts

```diff
@@ -443,6 +458,10 @@ OPTIONS:
         Do not prompt to update gitignore with generated code
     --future
         Include future features
+    --strictTypeNames
+        Return the exact string literal for .__type__.name instead of a general string type
+    --polymorphismAsDiscriminatedUnions
+        Use a discriminated union as the return type for polymorphic queries, where each member includes __typename
```

Comment:
```suggestion
    --future-strict-type-names
        Return the exact string literal for .__type__.name instead of a general string type
    --future-polymorphism-as-discriminated-unions
        Use a discriminated union as the return type for polymorphic queries, where each member includes __typename
```

Reply:
tnx, my brain doesn't work great after 8pm :D